### DISCUSSION TITLE
Disable "recent directory"?

### DISCUSSION THREAD

**Discussion by JavaliMZ** [marlonrichert/zsh-autocomplete#286] [Q&A] [answered]
Hello. It is possible to disable the "recent directory" section?

**Comment by marlonrichert** [comment, marked as answer]
It wasn't possible until a moment ago. 😉 But yes, now it is: e40c1318a30fd722f18ffa0ed99e20819ff60568

---

If you use my software on a regular basis, please consider [sponsoring me](https://github.com/sponsors/marlonrichert). 🙂
